app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/interactor/SubscriberDetailInteractorImpl.java

```diff
@@ -41,8 +46,13 @@ public void getUserRepositoryData(String userName, final OnFinishedListener list
         call.enqueue(new Callback<List<Repository>>() {
             @Override
             public void onResponse(Call<List<Repository>> call, Response<List<Repository>> response) {
-                List<Repository> repositoryList = response.body();
-                listener.onFinishedRepository(repositoryList);
+                if (response.isSuccessful()) {
+                    List<Repository> repositoryList = response.body();
+                    listener.onFinishedRepository(repositoryList);
+                } else {
+                    listener.onFailureRepository(Constants.MESSAGE_FAILED_SERVICE);
```

Comment:
This part will be to get better by Edwin, he will implement a best way for manage the errors in the next PR, because he started this task